v5/api/c/vision.rst

```diff
@@ -972,7 +967,7 @@ This function uses the following values of errno when an error state is
 reached:
 
 - ``ENXIO`` - The given value is not within the range of V5 ports (1-21).
-- ``ENODEV``  - The device cannot be configured as a vision sensor (e.g. it has been reconfigured)
+- ``EACCESS``  - Anothe resources is currently trying to access the port

```

Comment:
```suggestion
- ``EACCESS``  - Another resources is currently trying to access the port
```

Reply:
https://github.com/purduesigbots/pros/blob/ab99a3391c8d88129e479ff6ce450afe7e569d10/include/pros/vision.h#L539 but yeah this should be fixed either way